java/arcs/core/storage/DirectStoreMuxer.kt

```diff
@@ -75,11 +71,14 @@ class DirectStoreMuxer<Data : CrdtData, Op : CrdtOperation, T>(
 
     /** Calls [idle] on all existing contained stores and waits for their completion. */
     suspend fun idle() = storeMutex.withLock {
-        stores.values.map {
-            withContext(coroutineContext) {
-                launch { it.store.idle() }
-            }
-        }.joinAll()
+        stores.values.toList()
+    }.map {
+        /**
+         * If the overhead/wall-time of [DirectStore.idle] is longer than an
+         * [CoroutineScope.launch] i.e. more than 5ms debounce time, launching
+         * [DirectStore.idle]s in parallel can further help performance,
+         */
+        it.store.idle()
```

Comment:
Doesn't this need a launch {} to make it parallel? Or is this comment a TODO?

Reply:
Currently `store.idle` does nothing so `launch` here become negative performance impact.
So I put the comment there.